SourceX/DiabloUI/dialogs.cpp

```diff
@@ -196,6 +184,18 @@ void DialogLoop(UiItem *items, std::size_t num_items, UiItem *render_behind, std
 
 void UiOkDialog(const char *text, const char *caption, bool error, UiItem *render_behind, std::size_t render_behind_size)
 {
+	if (!gbActive) {
+		if (SDL_ShowCursor(SDL_ENABLE) <= -1) {
+			SDL_Log(SDL_GetError());
+		}
+		if (SDL_ShowSimpleMessageBox(SDL_MESSAGEBOX_ERROR, text, caption, NULL) <= -1) {

```

Comment:
I removed the stub `SDL_ShowSimpleMessageBox` for SDL1 in https://github.com/diasurgical/devilutionX/commit/4ea3a4e941bf48dd598932a2163d9db20772a86f so I think this won't compile for SDL1

Reply:
Thanks, I was unsure why this wasn't compiling, didn't think you had removed all usages.